src/sentry/tasks/sentry_apps.py

```diff
@@ -273,9 +274,24 @@ def send_webhooks(installation, event, **kwargs):
 
         request_data = AppPlatformEvent(**kwargs)
 
-        safe_urlopen(
+        resp = safe_urlopen(
             url=servicehook.sentry_app.webhook_url,
             data=request_data.body,
             headers=request_data.headers,
             timeout=5,
         )
+
+        # Track any webhook errors for these event types
+        if not resp.ok and event in ["issue.assigned", "issue.ignored", "issue.resolved"]:
```

Comment:
@nolachen what about errors on `error.created` and on installation/uninstallation?

Reply:
@scefali  talked about this with @MeredithAnya, I think for now we're just going with workflow webhook requests since there's the potential of creating a lot of errors for issue created/error created. Install/Uninstall could probably be added though

Reply:
@nolachen Not sure how I feel about this. By limiting the webhook requests we are handling, we are limiting the usefulness of this feature. But I can also see the performance implications of handling those cases. I suppose I am OK with this since this is a new feature. We can easily modify the code to handle those other cases in the future. But we should definitely make it clear on the UI that we are not catching errors for those events.